### PR TITLE
v92.4.0

### PR DESCRIPTION
**Pull request by andrewoverton**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,114 @@
...
+
+## New deprecations
+
+This minor release deprecates Chips+ChipThemer and MDCChipViewFontThemer.
+
+## New features
+
+MDCTextInputControllerOutlinedTextArea now allows users to specify `minimumLines` and `expandsOnOverflow`.
+
+```objc
```
Comment (codeman7) [inline CHANGELOG.md:13]: Should we add a Swift snippet?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,114 @@
...
+textInputControllerOutlinedTextArea.expandsOnOverflow = YES;
+```
+
+MDCBottomDrawerPresentationControllerDelegate has new methods that allow clients to respond to animation lifecycle events and touches to the scrim.
+
+#### Touch events set to delegate
+
+Touch events are propagated to delegate to allow clients to interpret touches in the scrim area.
+
+```
```
Comment (codeman7) [inline CHANGELOG.md:28]: Please document the language for this snippet, here and throughout.

**Review by codeman7 — COMMENTED** [review]
I believe for the changelog we should be calling out the deprecated methods and classes.

File: CHANGELOG.md (hunk 2 above)
Comment (andrewoverton) [reply CHANGELOG.md:28]: Done.

File: CHANGELOG.md (hunk 1 above)
Comment (andrewoverton) [reply CHANGELOG.md:13]: Done.

**Review by andrewoverton — COMMENTED** [review]
deprecations are listed under the New deprecations section.

**Review by codeman7 — APPROVED** [review]

**Pull request merged**